docker/storm/templates/Dockerfile.template

```diff
@@ -0,0 +1,122 @@
+{{- $storm := index .Packages "storm" -}}
+
+
+FROM marketplace.gcr.io/google/debian11:latest
+
+ARG STORM_VERSION="{{ $storm.Version }}"
+ENV C2D_RELEASE "{{ $storm.Version }}"
+ENV STORM_VERSION ${STORM_VERSION}
+
+ENV STORM_CONF_DIR=/conf \
+    STORM_DATA_DIR=/data \
+    STORM_LOG_DIR=/logs \
+    ZOO_CONF_DIR=/conf
+    
+
+
+# create Storm data directories
+
+RUN set -eux; \
+    groupadd -r storm --gid=1000 ;\
+    useradd -r -g storm --uid=1000 storm ;\
+    mkdir -p "$STORM_CONF_DIR" "$STORM_DATA_DIR" "$STORM_LOG_DIR"; \
+    chown -R storm:storm "$STORM_CONF_DIR" "$STORM_DATA_DIR" "$STORM_LOG_DIR"``
+
+# Add Java 11
+RUN apt-get update \
+  && apt-get install -y openjdk-11-jdk
+
+
+# Installing packags and dependencies
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+        python3-dev \
+        python3-pip \
+        openssl \
+        ca-certificates \
+        dirmngr \
+        gosu \
+        gnupg \
+        procps \
+        curl \
+        wget; \
```

Comment:
add openssl as well plz, just in case. :)

Reply:
openssl is already there)